authentication/src/main/java/io/deephaven/auth/MultiAuthorizationSource.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+ */
+package io.deephaven.auth;
+
+/**
+ * An AuthorizationSource that composes multiple AuthorizationSources together.
+ * <p>
+ * If an AuthorizationSource returns UNDECIDED, the next AuthorizationSource is consulted. If all AuthorizationSources
+ * return UNDECIDED, the result is UNDECIDED.
+ */
```

Comment:
It looks like it should be documented that _order_ matters - the first non-undecided result will be returned.